data/json/items/armor.json

```diff
@@ -5207,6 +5207,7 @@
     "warmth": 5,
     "material_thickness": 2,
     "container_data": { "contains": 40, "seals": true, "watertight": true },
+    "qualities": [ [ "BOIL", 1 ] ],
```

Comment:
Boil stuff in a plastic jerrycan?

Reply:
Oh, thought it was made from the metal one. Will remove.